### PR TITLE
reset world when lava is touched

### PR DESCRIPTION
**Pull request by erichsueh3**
Addressing issue #18

### CODE REVIEW

**Comment by erichsueh3** [comment]
This PR also addresses issue #26 by adding a negative reward for touching lava (dying).

File: agent.py [hunk 1]
```diff
@@ -150,11 +152,16 @@ def step(self, action):
...
         # Get Done
-        # Done is true if we reach max # of steps
+        # Done is true if we reach time limit
         done = False
         if self.episode_end - self.episode_start >= 30.0:
             done = True
             time.sleep(2)
+                    
+        # Done is also true if lava is stepped into
+        if (self.obs[0, int(self.obs_size/2)-1, int(self.obs_size/2)] == -1 and (command == 'move 1' or command == 'jumpmove 1')):
```
Comment (lilphunky) [inline agent.py:162]: elif here so it doesnt have to check this at the end of every episode

File: agent.py [hunk 2]
```diff
@@ -356,8 +363,14 @@ def get_mission_xml(self):
                             </RewardForCollectingItem>
                             <RewardForTouchingBlockType>
                                 <Block type="bedrock" reward="-1" />
+                                <Block type="lava" reward="-1" />
+                                <Block type="flowing_lava" reward="-1" />
```
Comment (lilphunky) [inline agent.py:367]: Dying i.e touching lava should probably be a greater negative reward than touching bedrock

File: agent.py (hunk 1 above)
Comment (erichsueh3) [reply agent.py]: Good catch

**Pull request merged**